google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -546,44 +546,24 @@ public Blob reload(BlobSourceOption... options) {
   }
 
   /**
-   * Updates the blob's information. Bucket or blob's name cannot be changed by this method. If you
-   * want to rename the blob or move it to a different bucket use the {@link #copyTo} and {@link
-   * #delete} operations. A new {@code Blob} object is returned. By default no checks are made on
-   * the metadata generation of the current blob. If you want to update the information only if the
-   * current blob metadata are at their latest version use the {@code metagenerationMatch} option:
-   * {@code newBlob.update(BlobTargetOption.metagenerationMatch())}.
+   * Updates the blob properties. The {@code options} parameter contains the preconditions for
```

Comment:
I'd state `{@code options}` is `BlobTargetOption` and instead of `preconditions` use `request options` and link to update API documentation: https://cloud.google.com/storage/docs/json_api/v1/objects/update

Reply:
> and link to update API documentation: https://cloud.google.com/storage/docs/json_api/v1/objects/update

done

> I'd state {@code options} is BlobTargetOption

I think this is clear from the parameter type

> and instead of preconditions use request options

Personally for me it was hard to understand the purspose of this parameter. So, I tried to rephrase this statement to help users not very familiar with the Storage details to get the meaning. I came up to:  `The options parameter contains the preconditions for  applying the update`

Reply:
Thanks, sgtm, I think having the link will help provide information.